cmd/kots/cli/pull.go

```diff
@@ -32,12 +32,13 @@ func PullCmd() *cobra.Command {
 			// strip it if included or else the rewrite images will fail
 
 			pullOptions := pull.PullOptions{
-				HelmRepoURI:         v.GetString("repo"),
-				RootDir:             ExpandDir(v.GetString("rootdir")),
-				Namespace:           v.GetString("namespace"),
-				Downstreams:         v.GetStringSlice("downstream"),
-				LocalPath:           ExpandDir(v.GetString("local-path")),
-				LicenseFile:         ExpandDir(v.GetString("license-file")),
+				HelmRepoURI: v.GetString("repo"),
+				RootDir:     ExpandDir(v.GetString("rootdir")),
+				Namespace:   v.GetString("namespace"),
+				Downstreams: v.GetStringSlice("downstream"),
+				LocalPath:   ExpandDir(v.GetString("local-path")),
+				LicenseFile: ExpandDir(v.GetString("license-file")),
+				// ConfigFile:          TODO: specify existing config file here
```

Comment:
What's the plan on this?

Reply:
this will be loaded from the folder created with the kots pull command